Update site for TauDEM 5.4.0

pkdash opened the pull request.

pkdash committed:
Add macOS Finder metadata to .gitignore

pkdash committed:
Update TauDEM version history to include 5.4.0 release and enhancements

pkdash committed:
Update links to David Tarboton's Hydrology Research Group website across multiple HTML files to reflect the new URL structure  - syncing Dave' local changes to repo

pkdash committed:
Add help.css file with styling for various elements and tables used in help documentation pages

pkdash committed:
Updating index.html to reflect changes in version 5.4.0

pkdash committed:
Add CatchHydroGeo.html for documentation of Catchment Hydrogeography tool

pkdash committed:
Add InunDepth.html for documentation of Inundation Depth Estimation tool

pkdash committed:
Update updates.html with version 5.4.0 details and modify logo links

pkdash committed:
Update downloads.html for TauDEM 5.4.0 release, including new download links and installation instructions

pkdash committed:
Updating all help related html files to mordern HTML syntax

- Simplified the HTML markup and replaced inline styles with a linked CSS stylesheet.
- Removed deprecated tags and attributes, ensuring compliance with modern HTML standards.

pkdash commented on TauDEM/taudem5/index.html:
The content of this file has changed - needs review.

pkdash commented on TauDEM/taudem5/support.html:
```diff
@@ -82,7 +79,7 @@
             <div class="content">
                 <h3>Need help</h3>
                 <p align="left">While there is no formal ongoing support for this freely distributed   open source software, I do do my best to respond to emails and help with problems as time permits. Feel free to contact me if you need to. It is most helpful if you include with your message screen shots of the error you are having and the input dialog immediately prior to occurrence of the erro ...
```
Probably we need to update this paragraph.

dtarb replied:
True - but leaving for now, with a small change to indicate that these are old suggestions

pkdash commented on TauDEM/taudem5/updates.html:
The content of this file has been updated - needs review.

dtarb replied:
reviewed

pkdash committed:
typo

pkdash commented on TauDEM/taudem5/downloads.html:
The content of this file has been updated - needs review.

dtarb replied:
I have reviewed. Please check the MPI version.  It was stated as 1#5.

pkdash replied:
That is the correct version of mpi I used for TauDEM development on Windows.

pkdash commented on TauDEM/taudem5/help54/CatchHydroGeo.html:
This is new file added - needs review.

dtarb replied:
I reviewed and updated it.  Could you please make an ArcGIS tool for this function and in the process investigate whether the ArcGIS documentation process makes it any easier to create the help files, or at least to get some consistency between ArcGIS help and TauDEM help.

pkdash replied:
Added this [issue](https://github.com/dtarb/TauDEM/issues/292) in TauDEM repo for creating the new ArcGIS tool.

pkdash commented on TauDEM/taudem5/help54/InunDepth.html:
This is a new file added - needs review.

dtarb replied:
Reviewed and updated. As with CatchHydroGeo could you please make and ArcGIS tool for this.

pkdash replied:
Added this [issue](https://github.com/dtarb/TauDEM/issues/292) in TauDEM repo for creating the new ArcGIS tool.

pkdash commented on TauDEM/versions.html:
The content of this file has been updated - needs review.

dtarb replied:
Good

pkdash commented on TauDEM/taudem5/help53/SpecializedGridAnalysisToolsetOverview.htm:
```diff
@@ -1,73 +1,98 @@
+                        <td class="description">Calculates the ratio of the natural log of the specific catchment area (contributing area) to slope, ln(a/S), or ln(a/tan (beta)). This provides an indication of depth to water table. No data values occur in locations where slope is 0 (flat).</td>
+                    </tr>
+                    <tr>
+                        <td class="parameter-name">Catchment Hydrogaphy</td>
```
Should the CatchHydroGeo tool be part of the Specialized Grid Analysis Toolset?

dtarb replied:
Yes.

pkdash commented on TauDEM/taudem5/help53/SpecializedGridAnalysisToolsetOverview.htm:
```diff
@@ -1,73 +1,98 @@
+                        <td class="description">Evaluates channel hydraulic properties based on HAND (Height Above Nearest Drainage) grid.</td>
+                    </tr>
+                    <tr>
+                        <td class="parameter-name">Inundation Depth Estimation</td>
```
Should the InunDepth tool be part of the Specialized Grid Analysis Toolset?

dtarb replied:
Yes

pkdash reviewed (commented):
@dtarb I have left some comments to make it bit easier for you to review. You don't need to review where the contents have not changed. Most of the file changes are formatting updates to use modern HTML syntax.

pkdash committed:
changing parameters table styles for improved layout and consistency

dtarb committed:
Improved tool descriptions for "Catchment Hydrography" to "Catchment Hydraulic Geometry" and "Inundation Depth Estimation" to "Inundation Depth" for clarity.
 Added detailed parameter explanations and examples in the documentation for "CatchHydroGeo" and "InunDepth" tools.
Edited installation instructions for Windows and Linux/MacOS.
Updated version notes for TauDEM 5.4.0 with release date and changes.
Updated copyright years from 2015 to 2026 in multiple HTML files.

dtarb committed:
Update documentation and license for TauDEM 5.4.0 release, including link corrections and copyright year adjustment

dtarb committed:
Add HAND Overview documentation and update tool links for clarity

pkdash committed:
Adding HAND tool category to toolbox overview documentation

pkdash committed:
Remove outdated descriptions for "Catchment Hydraulic Geometry" and "Inundation Depth" from Specialized Grid Analysis Toolset documentation

pkdash committed:
Update HAND tool descriptions

pkdash committed:
deleting taudem5.0 folder
